src/main/java/org/opensearch/security/configuration/DlsFlsValveImpl.java

```diff
@@ -264,7 +265,7 @@ public boolean invoke(PrivilegesEvaluationContext context, final ActionListener<
                 }
             }
 
-            if (request instanceof UpdateRequest) {
+            if (UpdateAction.NAME.equals(context.getAction())) {
```

Comment:
This change wouldn't be needed if https://github.com/opensearch-project/index-management/pull/1311 is merged